test/framework/integration_framework/fake_peer/block_storage.hpp

```diff
@@ -25,33 +26,28 @@ namespace integration_framework {
 
     class BlockStorage final {
      public:
-      using BlockPtr = std::shared_ptr<shared_model::proto::Block>;
+      using BlockPtr = std::shared_ptr<const shared_model::proto::Block>;
```

Comment:
https://www.fluentcpp.com/2019/01/22/pitfalls-of-aliasing-a-pointer-in-modern-cpp/
Please do not use generic `Ptr` suffix, as it can mean raw pointer, `shared_ptr`, and `unique_ptr`.